apps/prairielearn/elements/pl-sketch/pl-sketch.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
+
+window.PLSketchApi = {
+  sketchpad: {},
+  color: '#000000',
+  tool: null,
+
+  changeTool: function (toolName) {
+    if (toolName === this.tool) {
+      return;
+    }
+    console.log(toolName);
+    
+    if (toolName === 'pen') {
+      size = 5;
+      this.tool = 'pen';
+      $('#pen').css('background', '#C0C0C0');
+      $('#eraser').css('background', '#FAFAFA');
+      this.sketchpad.penSize = size;
+      this.sketchpad.color = this.color;
+    } else {
+      size = 30;
+      this.tool = 'eraser';
+      $('#eraser').css('background', '#C0C0C0');
+      $('#pen').css('background', '#FAFAFA');
+      this.sketchpad.penSize = size;
+      this.sketchpad.color = '#FFF';
+    }
+  },
+
+  updateColor: function (colorName) {
+    this.color = colorName;
+    $('.pl-sketch-color-icon').css('backgroundColor', this.color);
+    if (this.tool === 'pen') {
+      this.sketchpad.color = this.color;
+    }
+  },
+
+  setupSketchpad: function (width, height, uuid) {
+    console.log('#undo-'+uuid);
+    let element = '#pl-sketch-canvas-' + uuid;
+    var sketchpadObject = {};
+    sketchpadObject['width'] = width;
+    sketchpadObject['height'] = height;
+    sketchpadObject['element'] = element;
```

Comment:
```suggestion
    const sketchpadObject = { width, height, element };
```
There are other simplifications that can be done in other places in the code, such as using `const` instead of `let`, but this is one example.